common/models/workspace.js

```diff
@@ -215,6 +215,14 @@ module.exports = function(Workspace) {
             .filter(function(cc) { return cc.name != explorer; });
       }
 
+      // Remove legacyExplorer from LoopBack 3.x app template
```

Comment:
Yup, I agree. It would be harder to manage for versions beyond 3.x. Will change it.